convert2rhel/utils.py

```diff
@@ -663,25 +663,14 @@ def find_keyid(keyfile):
     return keyid.lower()
 
 
-def string_to_version(verstring):
-    """Return a tuple of (epoch, version, release) from a version string
-    This function was taken from softwarefactory-project/rdopkg
-    (https://github.com/softwarefactory-project/rdopkg/blob/1.4.0/rdopkg/utils/specfile.py)
-    """
-
-    # is there an epoch?
-    components = verstring.split(":")
-    if len(components) > 1:
-        epoch = components[0]
-        components.pop(0)
-    else:
-        epoch = "0"
+def set_locale():
```

Comment:
Looks like this snuck in during a rebase.  This function has recently been removed.

Reply:
Noted i'll remove it